src/App.tsx

```diff
@@ -1,13 +1,28 @@
-
-import "./App.css";
-import { MainScene } from "@/scenes";
-import { useThemeContext } from "./core/providers";
+import React from 'react';
+import { MainScene, MobileTabletScene } from '@/scenes';
+import { useThemeContext } from './core/providers';
+import './App.css';
 
 function App() {
-  const { theme } = useThemeContext()
+  const { theme } = useThemeContext();
+  const [width, setWidth] = React.useState(true);
```

Comment:
_width_ is something that can lead to mistakes, e.g. I don't know if it's a number or a flag, pixels?

Let's rename it to something like:

isSupportedScreenResolution

```diff
const [isSupportedScreenResolution, setIsSupportedScreenResolution] = React.useState(true);
```